models/games.js

```diff
@@ -3,7 +3,13 @@ const db = require('../db/connection.js');
 exports.selectCategories = () => {
     return db.query('SELECT * FROM categories;')
         .then(data => {
-            console.log(data.rows)
+            return data.rows;
+        })
+};
+
+exports.selectReviews = () => {
+    return db.query('SELECT * FROM reviews;')
```

Comment:
This returns all columns from the reviews table but the ticket asks for so this query needs work to get all the required columns.